.github/workflows/GA_CI.yml

```diff
@@ -0,0 +1,22 @@
+name: tests
+on: [push]
+
+    
+jobs:
+  run-test:
+    runs-on: ubuntu-latest
+    steps:
+      - run: sudo apt-get install poppler-utils
+      - uses: actions/checkout@v2
+      - name : Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
```

Comment:
Can we specify this to `>3.8` (if not possible, then just `3.8`)? We're using some stuff from Python 3.8, like walrus operator `:=` and so forth.